lib/gman_client.rb

```diff
@@ -8,4 +8,14 @@ def self.drivers
     respond = request.api.v1.drivers.get
     respond.map(&:to_h)
   end
+  def self.driver_commission_histories
```

Comment:
Is there a reason this is a module and not a class? I'm rather ignorant when it comes to writing gems.